src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -86,13 +92,12 @@ describe('<PillButton> component', () => {
             expect(getButtonByName(upArrowLabel)).toHaveClass(
                 'PillButton__button PillButton__button--isOpen'
             );
-            expect(getButtonByName('Pill name up arrow')).not.toHaveClass(
-                'PillButton__button--isOpen'
-            );
+            expect(screen.queryByText(upArrowLabel)).not.toBeInTheDocument();
```

Comment:
I'm unsure about this check. It is for the open state, so we should expect the arrow up to show. I think there is a problem with how you checking it. There is no text on the screen with the label. It is only a label :-)
I think you need to use `queryByLabelText` or with role and name here and also in the other tests in this document.
Then the arrow up should be in the document, and we can add a check that downArrowLabel isn't.

Also someone recently pointed out and it might be better to use `toBeVisible` then `toBeInTheDocument`.